hipe4ml/plot_utils.py

```diff
@@ -172,7 +178,7 @@ def plot_distr(list_of_df, column=None, figsize=None, bins=50, log=False, labels
         figsize = [20, 15]
 
     if labels is None:
-        labels = ['class{}'.format(i_class) for i_class, _ in enumerate(list_of_df)]
+        labels = [f'class{i_class}' for i_class, _ in enumerate(list_of_df)]
```

Comment:
Since this was the code smell, I would introduce a constant at the beginning of the module that contain the standard string to identify a class

Reply:
Not really sure how this is made to work with the fstring